boards/arm64/qemu/qemu-armv8a/src/qemu_virtio.c

```diff
@@ -0,0 +1,41 @@
+/****************************************************************************
+ * boards/arm64/qemu/qemu-armv8a/src/qemu_virtio.c
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/config.h>
+
+#include <syslog.h>
+#include <errno.h>
+#include <debug.h>
+
+#include <nuttx/board.h>
+#include <nuttx/virtio/virtio.h>
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+void arm64_netinitialize(void)
+{
+  virtio_init();
```

Comment:
it's better to move initialize out of arm64_netinitialize since other virtio driver also initialize in virito_init in the feature, I think.

Reply:
or let' call virtnet_init directly and let's virtnet_init call virtio_devinit instead.